apis/sgv2-restapi/src/main/java/io/stargate/sgv2/restapi/grpc/BridgeProtoValueConverters.java

```diff
@@ -27,15 +27,16 @@ public static BridgeProtoValueConverters instance() {
     return INSTANCE;
   }
 
-  public FromProtoConverter fromProtoConverter(List<QueryOuterClass.ColumnSpec> columns) {
+  public FromProtoConverter fromProtoConverter(
+      List<QueryOuterClass.ColumnSpec> columns, boolean optimizeMapData) {
```

Comment:
Instead of adding separate flag, would it make sense to pass a config object, as we may well get more options in future?
Maybe that's for future expansion if and when that is needed I guess.

Reply:
I'm working on this. Can we call the config object as `RequestConfig`  or `RequestParams` since it's per request?

Reply:
Made changes to pass `RequestParams` config object from API end points so that we can add fields in the future if required for any new requirements